integration_tests/run_pyspark_from_build.sh

```diff
@@ -348,6 +348,7 @@ EOF
             --driver-java-options "$driverJavaOpts" \
             $SPARK_SUBMIT_FLAGS \
             --conf 'spark.rapids.memory.gpu.allocSize='"$gpuAllocSize" \
+            --conf 'spark.executor.cores='"10" \
```

Comment:
this only takes care of TEST_PARALLEL=0

we could add somewhere towards the top  a default for `$PYSP_TEST_spark_executor_cores`
or you could just set it outside for the failing pipeline

Reply:
I've moved this out to work with parallel tests as well.